ouroboros-consensus/src/Ouroboros/Consensus/NodeNetwork.hs

```diff
@@ -195,12 +202,13 @@ consensusNetworkApps
        )
     => Tracer m (TraceSendRecv (ChainSync (Header blk) (Point blk)))
     -> Tracer m (TraceSendRecv (BlockFetch blk))
+    -> NodeParams m peer blk
     -> NodeKernel m peer blk
     -> ProtocolCodecs blk failure m bytesCS bytesBF
     -> ProtocolHandlers m peer blk
     -> NetworkApplication m peer bytesCS bytesBF ()
 
-consensusNetworkApps traceChainSync traceBlockFetch kernel
+consensusNetworkApps traceChainSync traceBlockFetch NodeParams{chainDB} kernel
```

Comment:
No need for `NodeParams`, use `getChainDB kernel`.

Reply:
done.